pkg/actions/commands.go

```diff
@@ -62,9 +62,8 @@ func Commands() {
 
 			Subcommands: []cli.Command{
 				{
-					Name:    "create",
-					Aliases: []string{""},
-					Usage:   "create a project on disk",
+					Name:  "create",
+					Usage: "create a project on disk",
```

Comment:
Nit: we have mismatch of capitals on "usage" fields, more seem to be capital than not though. I'd make all these capitalised

Reply:
changed all for project